### ISSUE TITLE
[USABILITY]: 526 v2 - Street address field should allow more than 20 characters

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1046]
## Issue
The street address field only allows 20 characters. The test user data has a string `3457 Virginia Oaks Dr` that is one character too long and throws an error when I press the Continue button. This could be problematic since we're pulling addresses from known data if available, and could leave users unable to enter their proper address for correspondence. Screenshot attached below.

## Acceptance Criteria
As a screen reader user, I'd like to be able to enter addresses longer than 20 characters into the street address field without it throwing an error.

## Environment
* MacOS Mojave
* Chrome latest
* https://staging.va.gov/disability-benefits/apply/form-526-all-claims/contact-information
* Test user

---
![screen shot 2019-02-06 at 8 22 24 am](https://user-images.githubusercontent.com/934879/52348174-4dbf1d00-29e9-11e9-98d6-3d1e188821d3.png)

**Comment by alexpappasoddball** [comment]
@YanaRoy & @AZSchneider I believe this one is for your team.  If you agree move to your backlog and remove "triage" and "triage-incident"

**Comment by SMLuthi** [comment]
@1Copenut @alexpappasoddball The 20 max length is due to an upstream limitation. More specifically, the EVSS submit endpoint address fields only allow max of 20 characters. Theres not much that can be done about that.

**Comment by 1Copenut** [comment]
@SMLuthi Copy that. I happened to notice it when the test address threw an error, and figured it was worth a mention. I have seen addresses that include directional indicators like NW or SE and proper names that could easily exceed this limit.

Is there a fallback or graceful handling if the user's address is too long? I'll close out the ticket if this is a non-issue/solved problem.

Thank you.

/cc @alexpappasoddball

**Comment by ndsprinkle** [comment]
After meeting with Silvio and Cameron yesterday, I have finally reached a decision on the best solution to this issue.

We discussed trying to automatically trim the address at 20 characters and move part of it to the next address line (there are three address lines in the 526 form), but we think this would introduce too many potential issues with addresses that are already multiple lines.

The best solution to the problem we can think of is drawing attention to the field as soon as the veteran reaches the page. This will most likely be done through the normal validation error, except the veteran will know as soon as they reach the page instead of having to click continue first.

I may create another ticket for this work since the original issue outlined above can't be solved under our current circumstances with the EVSS endpoint. If I create a new ticket, I will link it in a comment below.

**Comment by ndsprinkle** [comment]
Ticket created: https://github.com/department-of-veterans-affairs/va.gov-team/issues/5225

**Comment by 1Copenut** [comment]
@ndsprinkle Would you mind updating this entry in the [Accessibility Product List](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-product-review-list.md) with your new ticket number? It will help the accessibility specialists maintain an accurate count.
